app/src/main/java/de/westnordost/streetcomplete/screens/main/bottom_sheet/CreateNoteFragment.kt

```diff
@@ -140,7 +142,11 @@ class CreateNoteFragment : AbstractCreateNoteFragment() {
     override fun onComposedNote(text: String, imagePaths: List<String>) {
         /* pressing once on "OK" should first only close the keyboard, so that the user can review
            the position of the note he placed */
-        if (contentBinding.noteInput.hideKeyboard() == true) return
+        val rootInsets = ViewCompat.getRootWindowInsets(contentBinding.noteInput)
+        if (rootInsets?.isVisible(WindowInsetsCompat.Type.ime()) == true) {
```

Comment:
I think this should be a extension function, too then. `fun View.isKeyboardVisible(): Boolean`